### ISSUE TITLE
Logement

### ISSUE THREAD

**Issue by fjg** [betagouv/ecolab-data#22]
## Chauffage

"Quelle est votre consommation de bois en granulés ?"

=> préciser le temps (/an ou /mois)
=> Impossible de répondre en kWh, demander en kg ou euros

**Comment by laem** [comment]
Autres questions à poser de plus en plus : 

- [ ] pompe à chaleur
- [ ] clim

**Comment by laem** [comment]
Autre point : nous pouvons faire une simulation sans facture en utilisant les facteurs d'émissions moyens.

Pour aller plus loin (ce serait l'idéal), on pourrait aussi demander la température de chauffage l'hiver.De 17 à 24°C par exemple. 

C'est d'ailleurs une belle fiche action, voir dans notre base d'actions si nous avons un modèle pour estimer cette réduction, je crois que c'est le cas.

**Comment by martinregner** [comment]
Le sujet des PAC est revenu à plusieurs reprise en ce moment.
@laem @Benjamin-Boisserie-ABC que pensez-vous d'ajouter une carte PAC dans le choix du chauffage, même si derrière c'est le modèle "élec" qui s'applique ?

**Comment by laem** [comment]
C'est fait pour les notifs :)

**Comment by Clemog** [comment]
Commentaire intéressant reçu par mail :
> J'ai fait votre test.. petit problème j'ai une chaudière à granulés et un poêle à bois et je ne peux pas mettre les 2.

J'ai ajouté la suggestion dans la liste des tâches

A mon avis cette issue est importante au vu des problématique énergétiques de cet hiver

**Comment by laem** [comment]
> A mon avis cette issue est importante au vu des problématique énergétiques de cet hiver

Oui, à mon avis un focus dédié au chauffage serait nécessaire :+1:. C'est clairement le sujet du moment.
